### PR TITLE
[skip lint] Let smithy register all recipes in github

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by ocefpaf** [comment]
LGTM

File: .travis_scripts/create_feedstocks.py [hunk 1]
```diff
@@ -157,7 +154,10 @@ def write_token(name, token):
         # Break the previous loop to allow the TravisCI registering to take place only once per function call.
         # Without this, intermittent failures to synch the TravisCI repos ensue.
         # Hang on to any CI registration errors that occur and raise them at the end.
```
Comment (jakirkham) [inline .travis_scripts/create_feedstocks.py]: Should we refresh this comment a bit?

**Comment by jakirkham** [comment]
LGTM. Thanks @isuruf. Small suggestion above, but not blocking.

File: .travis_scripts/create_feedstocks.py (hunk 1 above)
Comment (isuruf) [reply .travis_scripts/create_feedstocks.py]: This doesn't change here right?

**Pull request merged**